packages/next/lib/load-custom-routes.ts

```diff
@@ -28,6 +44,7 @@ export type Redirect = Rewrite & {
 }
 
 export const allowedStatusCodes = new Set([301, 302, 303, 307, 308])
+const allowedHasTypes = new Set(['header', 'cookie', 'query', 'host'])
```

Comment:
`host` isn't mentioned in the docs, intended?

Reply:
Oh good catch, will add to docs!

Reply:
Added